src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/config/KafkaTopicsGrpcConfig.java

```diff
@@ -0,0 +1,416 @@
+    private String networkControlResponseTopic;
+
+    @Value("${kafka.topic.speaker.network.priv.region:kilda.network.control.response.priv.region}")
+    private String networkControlResponseRegionTopic;
+
+    @Value("${kafka.topic.speaker.flow.ping:kilda.speaker.flow.ping.storm}")
+    private String speakerFlowPingTopic;
+
+    @Value("${kafka.topic.speaker.flow.ping.region:kilda.speaker.flow.ping}")
+    private String speakerFlowPingRegionTopic;
+
+    @Value("${kafka.topic.grpc.speaker:kilda.grpc.speaker}")
+    private String grpcSpeakerTopic;
+
+    @Value("${kafka.topic.ping:kilda.ping.storm}")
+    private String pingTopic;
+
+    @Value("${kafka.topic.ping.region:kilda.ping}")
+    private String pingRegionTopic;
+
+    @Value("${kafka.topic.stats:kilda.stats.storm}")
+    private String statsTopic;
+
+    @Value("${kafka.topic.stats.region:kilda.stats}")
+    private String statsRegionTopic;
+
+    @Value("${kafka.topic.topo.disco:kilda.topo.disco.storm}")
+    private String topoDiscoTopic;
+
+    @Value("${kafka.topic.topo.disco.region:kilda.topo.disco}")
+    private String topoDiscoRegionTopic;
+
+    @Value("${kafka.topic.topo.nbworker:kilda.topo.nb.storm}")
+    private String topoNbTopic;
+
+    @Value("${kafka.topic.topo.nbworker.region:kilda.topo.nb}")
+    private String topoNbRegionTopic;
+
+    @Value("${kafka.topic.topo.reroute:kilda.topo.reroute.storm}")
+    private String topoRerouteTopic;
+
+    @Value("${kafka.topic.topo.isl.status:kilda.network.isl.status.priv}")
+    private String networkIslStatusTopic;
+
+    @Value("${kafka.topic.topo.switch.manager.nb:kilda.nb.swmanager_request.priv}")
+    private String topoSwitchManagerNbTopic;
+
+    @Value("${kafka.topic.topo.switch.manager.network:kilda.network.swmanager_request.priv}")
+    private String topoSwitchManagerNetworkTopic;
+
+    @Value("${kafka.topic.topo.switch.manager.nbworker:kilda.nbworker.swmanager_request.priv}")
+    private String topoSwitchManagerNbWorkerTopic;
+
+    @Value("${kafka.topic.topo.switch.manager:kilda.topo.switch.manager.storm}")
+    private String topoSwitchManagerTopic;
+
+    @Value("${kafka.topic.topo.switch.manager.region:kilda.topo.switch.manager}")
+    private String topoSwitchManagerRegionTopic;
+
+    @Value("${kafka.topic.topo.isl.latency:kilda.topo.isl.latency.storm}")
+    private String topoIslLatencyTopic;
+
+    @Value("${kafka.topic.topo.isl.latency.region:kilda.topo.isl.latency}")
+    private String topoIslLatencyRegionTopic;
+
+    @Value("${kafka.topic.topo.router.connected.devices.storm:kilda.connected.devices.priv.storm}")
+    private String topoConnectedDevicesTopic;
+
+    @Value("${kafka.topic.topo.floodlight.connected.devices.region:kilda.floodlight.connected.devices.priv}")
+    private String topoConnectedDevicesRegionTopic;
+
+    @Value("${kafka.topic.topo.history.storm:kilda.topo.history.storm.priv}")
+    private String topoHistoryTopic;
+
+    // TODO(surabujin): check usage
```

Comment:
maybe create an issue for it? Seems like nobody remembers that this TODO exists.